GoodReads Series Sync Lists Showing Zero Results For Valid List IDs

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Goodreads Series Lists are returning zero results when List Sync is ran.

- Test Passing for List
`2022-02-26 06:47:15.5|Trace|Http|Req: 144 [POST] /api/v1/importlist/test (from 192.168.1.201)
2022-02-26 06:47:15.5|Debug|GoodreadsProxy|Getting Series with GoodreadsId of 299373
2022-02-26 06:47:15.5|Trace|CachedHttpResponseService|Returning cached response for [GET] https://www.goodreads.com/series/299373?key=dcX1YH52UMjs6lznO2MuSg&_nc=1&format=xml
2022-02-26 06:47:15.5|Trace|EventAggregator|Publishing ProviderStatusChangedEvent<IImportList>
2022-02-26 06:47:15.5|Trace|Http|Res: 144 [POST] /api/v1/importlist/test: 200.OK (10 ms)
2022-02-26 06:47:15.5|Debug|Api|[POST] /api/v1/importlist/test: 200.OK (10 ms)`


- Initiating Sync with List that passed test. Results show zero list items, though the returned XML document has 6 books listed.
`2022-02-26 06:48:45.1|Info|ImportListSyncService|Starting Import List Sync
2022-02-26 06:48:45.1|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-02-26 06:48:45.1|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-02-26 06:48:45.1|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-02-26 06:48:45.1|Debug|FetchAndParseImportListService|Available import lists 1
2022-02-26 06:48:45.1|Debug|GoodreadsProxy|Getting Series with GoodreadsId of 299373
2022-02-26 06:48:45.1|Trace|CachedHttpResponseService|Returning cached response for [GET] https://www.goodreads.com/series/299373?key=dcX1YH52UMjs6lznO2MuSg&_nc=1&format=xml
2022-02-26 06:48:45.1|Trace|EventAggregator|Publishing ProviderStatusChangedEvent<IImportList>
2022-02-26 06:48:45.1|Debug|FetchAndParseImportListService|Found 0 from Goodreads Series
2022-02-26 06:48:45.1|Debug|FetchAndParseImportListService|Found 0 reports
2022-02-26 06:48:45.1|Info|ImportListSyncService|Processing 0 list items
2022-02-26 06:48:45.1|Trace|EventAggregator|Publishing CommandUpdatedEvent
2022-02-26 06:48:45.1|Trace|EventAggregator|CommandUpdatedEvent -> CommandController
2022-02-26 06:48:45.1|Trace|EventAggregator|CommandUpdatedEvent <- CommandController
2022-02-26 06:48:45.1|Debug|AuthorMetadataRepository|0 author metadata up to date; Updating 0, Adding 0 author metadata entries.
2022-02-26 06:48:45.1|Trace|EventAggregator|Publishing AuthorsImportedEvent
2022-02-26 06:48:45.1|Trace|EventAggregator|AuthorsImportedEvent -> AuthorAddedHandler
2022-02-26 06:48:45.1|Trace|EventAggregator|AuthorsImportedEvent <- AuthorAddedHandler`

### Expected Behavior

Expecting Goodreads Series Lists to return series info and listed books on Sync List task, (if no current Author) importing new Author, marking books from list as Wanted per List settings.

### Steps To Reproduce

In Settings / Import Lists, add new list of type Goodreads Series. 
Use valid series ID, IE 299373
Save list.
In System / Tasks run Import List Sync task
View results

### Environment

```markdown
Version: 0.1.1.1259
Package Version: nightly-9b5c3ee by hotio
.NET Core: Yes (6.0.2)
Docker: Yes
DB Migration: 21
```


### What branch are you running?

Nightly

### Trace Logs?

The reason for the other lists all failing and being skipped is they all returned zero results in the last sync, but Testing a list clears its "Failed" status.

[TestList.txt](https://github.com/Readarr/Readarr/files/8146642/TestList.txt)
[SyncLists.txt](https://github.com/Readarr/Readarr/files/8146643/SyncLists.txt)

Debugging this

work is missing the bestbook

```
work
error CS0103: The name 'DebuggerDisplay' does not exist in the current context
    AverageRating: 0
    BestBook: null
    BestBookId: null
    BooksCount: 0
    ElementName: "work"
    Id: 1491678
    MediaType: null
    OriginalLanguageId: null
    OriginalPublicationDate: null
    OriginalTitle: null
    RatingDistribution: null
    RatingsCount: 0
    ReviewsCount: 0
    SeriesLinkId: 0
    TextReviewsCount: 0
    UserPosition: null
```
at https://github.com/Readarr/Readarr/blob/20b1f41ac1569e6f8a477a1e762a5317e9db80a4/src/NzbDrone.Core/ImportLists/Goodreads/Series/GoodreadsSeriesImportList.cs#L46

yet the xml does have it; seems like a resource issue @ta264 ?

```xml
<work>
<id>72407681</id>
<uri>kca://work/amzn1.gr.work.v2.1fef3019-6c7e-4b8e-bf6e-0a96d055da71</uri>
<best_book>
<id>52393324</id>
<title>Ruins of the Galaxy (Ruins of the Galaxy #1)</title>
<author>
<id>13828892</id>
<name>J.N. Chaney</name>
</author>
<image_url>
<![CDATA[ https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1564303290l/52393324._SX98_SY160_.jpg ]]>
</image_url>
</best_book>
<books_count>3</books_count>
<original_publication_day>27</original_publication_day>
<original_publication_month>7</original_publication_month>
<original_publication_year>2019</original_publication_year>
<original_title>Ruins of the Galaxy</original_title>
<ratings_count>1160</ratings_count>
<ratings_sum>4804</ratings_sum>
<reviews_count>5120</reviews_count>
<text_reviews_count>86</text_reviews_count>
<average_rating/>
<policy_tags> </policy_tags>
<feature_flags> </feature_flags>
</work>
```

--- a/src/NzbDrone.Core.Test/MetadataSource/Goodreads/Resources/WorkResourcesFixture.cs
+++ b/src/NzbDrone.Core.Test/MetadataSource/Goodreads/Resources/WorkResourcesFixture.cs
@@ -1,0 +1,62 @@
+using System;
+using System.Xml.Linq;
+using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Core.MetadataSource.Goodreads;
+using NzbDrone.Core.Test.Framework;
+using NzbDrone.Test.Common;
+
+namespace NzbDrone.Core.Test.MetadataSource.Goodreads.Resources
+{
+    [TestFixture]
+    public class WorkResourceFixture : CoreTest<WorkResource>
+    {
+        [Test]
+        public void parse_non_work()
+        {
+            XElement element = new XElement("Dummy", "entry");
+            WorkResource work = new WorkResource();
+
+            Assert.Throws<NullReferenceException>(() => work.Parse(element));
+
+            work.OriginalTitle.Should().Be(null);
+
+            ExceptionVerification.IgnoreWarns();
+        }
+
+        [Test]
+        public void parse_minimal_work()
+        {
+            XElement element = new XElement("work",
+                new XElement("original_title", "Book Title"),
+                new XElement("id", "123456789"));
+
+            WorkResource work = new WorkResource();
+
+            work.Parse(element);
+
+            work.OriginalTitle.Should().Be("Book Title");
+            work.Id.Should().Be(123456789);
+
+            ExceptionVerification.IgnoreWarns();
+        }
+
+        [Test]
+        public void parse_minimal_work_with_surrounding_tags()
+        {
+            XElement element = new XElement("series_works",
+                new XElement("work",
+                    new XElement("original_title", "Book Title"),
+                    new XElement("id", "123456789")));
+
+            WorkResource work = new WorkResource();
+
+            work.Parse(element);
+
+            work.OriginalTitle.Should().Be("Book Title");
+            work.Id.Should().Be(123456789);
+
+            ExceptionVerification.IgnoreWarns();
+        }
+    }
+}

--- a/src/NzbDrone.Core/MetadataSource/Goodreads/Resources/WorkResource.cs
+++ b/src/NzbDrone.Core/MetadataSource/Goodreads/Resources/WorkResource.cs
@@ -94,6 +94,11 @@ namespace NzbDrone.Core.MetadataSource.Goodreads
 
         public override void Parse(XElement element)
         {
+            if (element.AncestorsAndSelf().First().Name != "work")
+            {
+                element = element.Element("work");
+            }
+
             Id = element.ElementAsLong("id");
 
             var bestBookElement = element.Element("best_book");

